x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/field_mapping_row.tsx

```diff
@@ -54,16 +72,18 @@ const FieldMappingRowComponent: React.FC<RowProps> = ({ siemField, thirdPartyOpt
       </EuiFlexItem>
       <EuiFlexItem>
         <EuiSuperSelect
+          disabled={disabled}
           options={thirdPartyOptions}
           valueOfSelected={selectedThirdParty}
-          onChange={setSelectedThirdParty}
+          onChange={onChangeThirdParty.bind(null, siemField)}
         />
       </EuiFlexItem>
       <EuiFlexItem>
         <EuiSuperSelect
-          options={editUpdateOptions}
-          valueOfSelected={selectedEditUpdate}
-          onChange={setSelectedEditUpdate}
+          disabled={disabled}
+          options={actionTypeOptions}
```

Comment:
are we offering every `actionTypeOptions` for each field? For example, the comments wont be able to overwrite

Reply:
For the moment yes. But you are right, we should remove unsupported actions. Let's do that in another PR.

Reply:
make sure to make an issue so we don't forget to do this!

Reply:
https://github.com/elastic/kibana/issues/60056